src/Symfony/Component/OptionsResolver/Tests/OptionsResolver2Dot6Test.php

```diff
@@ -1557,4 +1522,57 @@ public function testCountFailsOutsideResolve()
 
         count($this->resolver);
     }
+
+    /**
+     * @dataProvider getFormatValueData
+     * @expectedException \Symfony\Component\OptionsResolver\Exception\InvalidOptionsException
+     */
+    public function testFormatValue($actualType, $allowedType, $exceptionMessage)
+    {
+        $this->resolver->setDefined('option');
+        $this->resolver->setAllowedTypes('option', $allowedType);
+
```

Comment:
you need to use `setExpectedException` https://phpunit.de/manual/4.1/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.exceptions.examples.ExceptionTest3.php to test against the messge as well